### DISCUSSION TITLE
Operazioni multiple/massive

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#640] [Idee e proposte]
Buonasera ,mi ritrovo qui ,in quanto  ieri dovevo mettere a tutta la  classe il "fuoriclasse" perché occupata con un'altra attività .Ebbene sì ,ho riscontrato un enorme problema ,ovvero, ho dovuto inserire singolarmente il fuoriclasse alunno per alunno perdendo un sacco di tempo , quando invece negli altri registri vi è la cosiddetta "evento massivo/multiplo" che risolve il tutto selezionando tutti gli alunni  con un unico clic in una celletta e questo vale sia per i seguenti eventi :assenza, ritardo,uscita,fuoriclasse .Spero sia implementato nella prossima versione

**Comment by trinko** [comment]
Dalla pagina **coordinatore > fuori classe** puoi inserire un intervallo di date e selezionare più alunni.
L'unica cosa che manca, in effetti, è il pulsante seleziona tutti.

**Comment by sempliciniluigi89** [comment]
Ma credo sarebbe più utile se la funzione fosse estesa a tutti i docenti
facenti parte del consiglio e non solo al coordinatore ,perché nel momento
in cui tutta la classe è impegnata in altre attività io docente  di quella
fascia oraria devo poter mettere in modo massivo/multiplo  il fuoricalsse a
tutti ,senza dover contattare il coordinatore e chiedere di svolgere tale
azione devo poter mettere

Il giorno lun 29 gen 2024 alle ore 00:02 Antonello Dessì <
***@***.***> ha scritto:
